[2024-06-24T10:12:43.747+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T10:12:43.769+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.process_data_kyiv scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-24T10:12:43.779+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.process_data_kyiv scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-24T10:12:43.779+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-06-24T10:12:43.796+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): processing_tasks.process_data_kyiv> on 2024-06-23 00:00:00+00:00
[2024-06-24T10:12:43.803+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=130) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T10:12:43.805+0000] {standard_task_runner.py:63} INFO - Started process 134 to run task
[2024-06-24T10:12:43.806+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'processing_tasks.process_data_kyiv', 'scheduled__2024-06-23T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpqduy42s4']
[2024-06-24T10:12:43.808+0000] {standard_task_runner.py:91} INFO - Job 55: Subtask processing_tasks.process_data_kyiv
[2024-06-24T10:12:43.857+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_dag.processing_tasks.process_data_kyiv scheduled__2024-06-23T00:00:00+00:00 [running]> on host afc8096e8b86
[2024-06-24T10:12:43.969+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='processing_tasks.process_data_kyiv' AIRFLOW_CTX_EXECUTION_DATE='2024-06-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-23T00:00:00+00:00'
[2024-06-24T10:12:43.971+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T10:12:43.983+0000] {python.py:237} INFO - Done. Returned value was: (1719223721, 293.9)
[2024-06-24T10:12:43.984+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T10:12:44.009+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=processing_tasks.process_data_kyiv, run_id=scheduled__2024-06-23T00:00:00+00:00, execution_date=20240623T000000, start_date=20240624T101243, end_date=20240624T101244
[2024-06-24T10:12:44.060+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T10:12:44.070+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
